yarn-project/circuits.js/src/cbind/cbind.ts

```diff
@@ -0,0 +1,91 @@
+import { CircuitsWasm } from '../wasm/index.js';
+import { decode, encode } from '@msgpack/msgpack';
+
+/**
+ * Recursively converts Uint8Arrays to Buffers in the input data structure.
+ * The function traverses through the given data, and if it encounters a Uint8Array,
+ * it replaces it with a Buffer. It supports nested arrays and objects.
+ *
+ * @param data - The input data structure that may contain Uint8Arrays.
+ * @returns A new data structure with all instances of Uint8Array replaced by Buffer.
+ */
+function recursiveFixDecoded(data: any): any {
```

Comment:
Consider renaming function.  I understand what the function is doing, but I do not understand the name.